devel/Makefile

```diff
@@ -1,5 +1,9 @@
+THIS_FILE := $(lastword $(MAKEFILE_LIST))
+THIS_DIR := $(shell realpath $(dir $(THIS_FILE)))
+
 all:
-	./run-tests
+	@chmod a+w "$(THIS_DIR)/../devel"
+	sudo -u nobody ./run-tests
```

Comment:
I meant do this in the Dockerfile. :)

Or "sudo make all" if you really want to go via the Makefile.